.github/workflows/integration-tests.yml

```diff
@@ -30,8 +30,8 @@ jobs:
           distribution: 'temurin'
           cache: 'gradle'
 
-      - name: Gradle build service
-        run: ./gradlew --build-cache :service:build -x test
+      - name: Build without running tests
```

Comment:
Do we have an equivalent test runner job? I see the test flag still there?

Reply:
Not sure what you're asking.. although now that I think about it, I don't know why this builds `service` at all; it should really be building the integration subproject, since this build script doesn't run the service at all. I also don't know why this calls `jib`. I'll make a JIRA to track the cleanup of this code.

Reply:
Not sure what you're asking.. although now that I think about it, I don't know why this builds `service` at all; it should really be building the integration subproject, since this build script doesn't run the service at all. I also don't know why this calls `jib`. I'll make a JIRA to track the cleanup of this code.

I think the idea here is to separate build errors from test errors by doing this, but for the integration test run there's probably not much value in doing it this way.

Reply:
I was just noticing the name change here and wondering what was different/why. If you have a good sense of the cleanup, that works.